### ISSUE TITLE
24net pts.imdb

### ISSUE THREAD

**Issue by Zouhj** [CongWeilin/mtcnn-caffe#36]
Hi, Weilin, thank you for your share. I am confused about facail landmark dataset. It sames that we should use model trained by 12net to detect Celeba and get the training pts data for 24net. But from your code prepare_data, I don't find this. So could you please tell me how the pts data cascade?

**Comment by CongWeilin** [comment]
you can copy the 12net code for 24net

**Comment by Zouhj** [comment]
@CongWeilin Yeah, thank you for your respond. But I notice that the model provided is original trained by the author of MTCNN. Could you please share the model trained by your method. because my training result is not good, I want to test and analyse the reasons. Thank you so much, wish for your responding.

**Comment by wolf345** [comment]
@Zouhj Hi,zouhj.no 12/pts.imdb,How is your 12/ptsimdb file generated？
